test/torchaudio_unittest/librosa_compatibility_test.py

```diff
@@ -233,6 +233,17 @@ def assert_compatibilities(self, n_fft, hop_length, power, n_mels, n_mfcc, sampl
         self.assertEqual(
             torch_mfcc.type(librosa_mfcc_tensor.dtype), librosa_mfcc_tensor, atol=5e-3, rtol=1e-5)
 
+        # test SpectralCentroid
```

Comment:
Can you make this into a stand alone test method?
The code snippet above is legacy code and it's discouraged to perform multiple tests in one method.